src/migrations/20240208130439-events-revision-id-index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function(db, cb) {
+    db.runSql(
+        `
+            CREATE INDEX IF NOT EXISTS idx_events_feature_type_id ON events (id)
+                WHERE feature_name IS NOT NULL
+                    AND type IN ('segment-updated', 'feature_import', 'features-imported');
```

Comment:
The query is feature_name IS NOT NULL OR type IN (...), should the index also use OR here?

Reply:
Good catch, was testing different variants and typo appeared.